x/evm/types/params.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2021 Evmos Foundation
+// This file is part of Evmos' Evmos packages.
+//
+// The Evmos packages is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Evmos packages is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Evmos packages. If not, see https://github.com/evmos/ethermint/blob/main/LICENSE
+package types
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/params"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/evmos/ethermint/types"
+)
+
+var (
+	// DefaultEVMDenom defines the default EVM denomination on Ethermint
+	DefaultEVMDenom = types.AttoPhoton
```

Comment:
Nit: change to `aevmos`. You can use the [BaseDenom on types pkg](https://github.com/evmos/evmos/blob/1d4407b7a60e915f73f446101ddca09ed5b2915b/types/utils.go#L38)

Reply:
ditto